src/RMP/Translate/lang/programmes/en.po

```diff
@@ -1686,3 +1686,33 @@ msgstr[2] "%1 total"
 #. English: "You may also like".
 msgid "you_might_also_like"
 msgstr "You may also like"
+
+#. English "Now playing".
+#. Priority for: Rich Tracks
+msgid "now_playing"
+msgstr "Now Playing"
+
+#. English "s ago".
+#. Priority for: Rich Tracks
+msgid "seconds_ago_short"
+msgstr "%i%s ago"
```

Comment:
Also, would be nice to add an example variable for the translation tool, e.g. 

#. %1 is a variable, which may contain the value: "1".

The translate tool will then generate a preview of the finished string